person.rb

```diff
@@ -14,7 +14,7 @@ def initialize(age:, name: 'Unknown', parent_permission: true)
   # Getters & Setters - changed by rubocop
   attr_reader :id
 
-  attr_accessor :age, :name
+  attr_accessor :age, :name, :rentals
```

Comment:
- [ ] A person has many rentals, please 🙏🏽 initialize the instance variable `@rentals = [ ]` in your constructor to create the relationship between person and rental.